sum-of-nth-terms.py

```diff
@@ -0,0 +1,15 @@
+def series_sum(n):
```

Comment:
This filename of "sum-of-nth-terms" should be abolished permanently from your system. **NEVER AGAIN SHALL YE USE KABOB CASE FOR A PYTHON SCRIPT!**. I can't `import` anything from your code now because  of the hyphens.